### PR TITLE
Mongo switch over

### PR DESCRIPTION
**Pull request by johnosullivan**

### CODE REVIEW

File: serverless.yml [hunk 1]
```diff
@@ -66,15 +67,21 @@ functions:
...
   createArt:
     handler: index.handler
     events:
       - http: 'POST /art'
         cors: true
-    authorizer:
-        type: COGNITO_USER_POOLS
-        authorizerId:
-            Ref: ApiGatewayAuthorizer
+    #authorizer:
```
Comment (dconroy) [inline serverless.yml]: why is this commented out again?

File: serverless.yml [hunk 2]
```diff
@@ -66,15 +67,21 @@ functions:
...
+    #    type: COGNITO_USER_POOLS
+    #    authorizerId:
+    #        Ref: ApiGatewayAuthorizer
 
 resources:
   Resources:
```
Comment (dconroy) [inline serverless.yml]: after a successful migration we will need to update the serverless to blow away the dynamo

File: serverless.yml [hunk 3]
```diff
@@ -46,6 +46,7 @@ provider:
         - { "Fn::GetAtt": ["StreetArtDynamoDBTable", "Arn" ] }
   environment:
     ART_TABLE: ${self:custom.streetArtTableName}
+    MONGO_DB: mongodb+srv://streettagged:XrdG9JQShFmAy4rd@streettagged-z3s5t.mongodb.net/streettagged?retryWrites=true&w=majority
```
Comment (dconroy) [inline serverless.yml]: these look like secrets? or no? this is apublic repo

File: serverless.yml (hunk 1 above)
Comment (johnosullivan) [reply serverless.yml]: We make it public and once it work we add the permission

File: serverless.yml (hunk 3 above)
Comment (johnosullivan) [reply serverless.yml]: We have have some where to store it

File: serverless.yml (hunk 2 above)
Comment (johnosullivan) [reply serverless.yml]: Agreed once we have it updated we blow again the dynamo

**Pull request merged**